media-gfx/gif2apng/gif2apng-1.9.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="Converts GIF animations into APNG format"
+DESCRIPTION="This program converts GIF animations into animated PNG format. \
+Usually it makes the files smaller."
+HOMEPAGE="http://gif2apng.sourceforge.net/"
+COPYRIGHT="2009-2013 Max Stepin"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/gif2apng/gif2apng-$portVersion-src.zip"
+CHECKSUM_SHA256="3b21308e935d799b3ffb4a86c6e00ffa4cb9b3f72f52d58d51c66eb0574ae7d2"
+SOURCE_DIR=""
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gif2apng$secondaryArchSuffix = $portVersion
+	cmd:gif2apng = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:automake
+	cmd:ld$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	if [ "$targetArchitecture" != x86_gcc2 ]; then
+		commandBinDir=$binDir
+	else
+		commandBinDir=$prefix/bin
+	fi
+	mkdir -p $commandBinDir
+	cp gif2apng $commandBinDir
```

Comment:
Maybe we should simplify INSTALL to have only:
```
INSTALL()
{
	mkdir -p $prefix/bin
	cp gif2apng $prefix/bin
}
```
I had suggested defining commandBinDir because we often do that, but since we don't even use it in BUILD (for this recipe) it does not really make sense to define commandBinDir just for INSTALL. Moreover, we just use it twice. (So sorry for the bad suggestion...)